instance/models/openedx_instance.py

```diff
@@ -274,9 +274,14 @@ def get_load_balancer_configuration(self):
         """
         if not self.active_appserver:
             return self.get_preliminary_page_config(self.ref.pk)
+        ip_address = self.active_appserver.server.public_ip
+        if not ip_address:
+            # The active appserver doesn't have a public IP address.  This means that the server
+            # has been terminated, so we don't show the preliminary page in this case, and simply
+            # deconfigure the backend instead.
+            return [], []
```

Comment:
But this _isn't_ an error.  It just means the active appserver has been terminated, which is what happens if an instance gets shut down.